lambdas/update_service_list/update_service_list.py

```diff
@@ -127,9 +129,13 @@ def main(event, _):
         [create_boto_client('ecs', role_arn) for role_arn in assumable_roles]
     ) + [boto3.client('ecs')]
 
-    cluster_lists = (
-        [get_cluster_list(ecs_client) for ecs_client in ecs_clients]
-    )
+    try:
+        cluster_lists = (
+            [get_cluster_list(ecs_client) for ecs_client in ecs_clients]
+        )
```

Comment:
minor: you can move the square brackets in place of the parentheses, and I think it looks a bit cleaner, i.e.

```python
cluster_lists = [
    get_cluster_list(ecs_client) for ecs_client in ecs_clients
]
```